test/e2e/tests/network/add-custom-network.spec.js

```diff
@@ -606,7 +607,8 @@ describe('Custom network', function () {
           await driver.clickElement(
             '[data-testid="account-options-menu-button"]',
           );
-          await driver.clickElement({ text: 'Settings', tag: 'div' });
+          await driver.waitForSelector('[data-testid="global-menu-settings"]');
+          await driver.clickElement('[data-testid="global-menu-settings"]');
           await driver.clickElement({ text: 'Networks', tag: 'div' });
```

Comment:
Should we just add timeout here since `clickElement` has `findElement` inside?
```
 const element = await this.findClickableElement(rawLocator);
        await element.click();
```

Reply:
Thanks, @DDDDDanica. That's a valid point about adding the timeout. This issue is happening only for the mmi build, and the menu item is changing. So, I have written a conditional wait for the menu in the code below. Let me know your thoughts.

```
if (process.env.METAMASK_BUILD_TYPE === 'mmi') {
              await driver.wait(async () => {
                const element = await driver.findVisibleElement(
                  '[data-testid="global-menu-mmi-portfolio"]',
                );
                if (!element) {
                  console.log('Element not visible');
                }
              }, 1000);
        }
```